editor/views/editoritem.py

```diff
@@ -511,10 +511,51 @@ def get(self, *args, **kwargs):
 
     def post(self, request, *args, **kwargs):
         ei = self.get_object()
+        error_message = ''
+        count = 1
+        if (ei.item_type == 'exam'):
+            exam_questions = ei.exam.newexamquestion_set.all().order_by('qn_order').distinct()
+            for eq in exam_questions:
+                question = eq.question
+                if question.editoritem.published == True:
```

Comment:
You could add `.exclude(published=True)` to your query for `exam_questions` to filter out questions that have already been published.